src/Nuget.Dockerfile

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+FROM mcr.microsoft.com/dotnet/aspnet:8.0.3-alpine3.19-amd64@sha256:a531d9d123928514405b9da9ff28a3aa81bd6f7d7d8cfb6207b66c007e7b3075 AS runtimebase
+
+RUN apk --no-cache add curl=~8 unzip=~6 dos2unix=~7 bash=~5 gettext=~0
```

Comment:
what is the purpose of installing `curl`, `dos2unix`, `bash`, and `gettext`?

Reply:
Since I won't be here to respond tomorrow, I'll go ahead and partially answer my question.

* I have seen `curl` in some other container, but we don't need it: we have `wget`.
* `dos2unix` is used to change the line types on files we copy in, but we don't need it in our case. The run script and appsettings template are already saved in Git with the correct line endings (`\lf` without `\cr`).
* Alpine doesn't come with `bash` and that makes it a little harder to debug. For localhost development we definitely prefer Bash. But it does have `sh`, which is good enough. So I recommend skipping `bash` for this "production level image".
* I don't even remember what `gettext` is, and I don't see us using it anywhere here.

Reply:
- Removed `curl, dos2unix`
- `gettext` is required for this line in the sh file `envsubst < /app/appsettings.template.json > /app/appsettings.json`
- `bash` was removed